src/components/PetitionList/PetitionList.tsx

```diff
@@ -1,129 +1,33 @@
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-      <div>hello world!</div>
-    </>
+    <div className={cx('petition_list')}>
+      <div className={cx('petition_new')}>최신 청원들</div>
+      <div className={cx('petition_write')}>
+        <a href="/petition/write">지금 청원하기</a>
+      </div>
+      <div className={cx('petition_cards')}>
+        <div className={cx('petition_header')}>
+          <span className={cx('category')}>카테고리</span>
+          <span className={cx('title')}>제목</span>
+          <span className={cx('end')}>청원마감일</span>
+          <span className={cx('agree-num')}>참여인원</span>
+        </div>
+        {mockData.map(data => {
+          return <PetitionCard key={data.id} data={data} />;
+        })}
```

Comment:
```suggestion
        {mockData.map(petition => {
          return <PetitionCard key={data.id} petition={petition} />;
        })}
```
속성명을 petition으로 바꾸면 어떨까요? 그럼 PetitonCard에서 사용할때도 좀더 직관적일것 같습니다

Reply:
오 맞네요!!